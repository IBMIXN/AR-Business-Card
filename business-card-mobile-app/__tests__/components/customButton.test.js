import React from "react";
import renderer from "react-test-renderer";
import CustomButton from "../../components/customButton";

// jest.mock("react-native/Libraries/Animated/src/NativeAnimatedHelper");

describe("<CustomButton />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<CustomButton title={"Testing"} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
