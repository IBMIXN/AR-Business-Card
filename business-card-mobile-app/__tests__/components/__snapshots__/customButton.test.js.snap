// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomButton /> renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#0043CE",
      "borderRadius": 20,
      "borderWidth": 2,
      "opacity": 1,
      "padding": 10,
      "width": "48%",
    }
  }
>
  <View>
    <Text
      style={
        Object {
          "alignSelf": "center",
          "color": "white",
          "fontSize": 16,
        }
      }
    >
      Testing
    </Text>
  </View>
</View>
`;
